bench/workgen/runner/checkpoint-latency.wtperf

```diff
@@ -0,0 +1,26 @@
+# A stress configuration to create checkpoints while doing a mix of inserts
```

Comment:
I had put this .wtperf file in the workgen directory because it used an option (`sample_file`) that was not available with the standard wtperf.  I just pushed a better resolution that creates a `"monitor.json"` file whenever a `"monitor"` file is created (that is, when `sample_interval` is used).  Then the incompatible option can be removed, and this file can be moved into the `wtperf/runners` directory as suggested.